src/core/layout/qgslayoutaligner.cpp

```diff
@@ -97,6 +99,65 @@ void QgsLayoutAligner::distributeItems( QgsLayout *layout, const QList<QgsLayout
   if ( items.size() < 2 )
     return;
 
+  double minCoord = std::numeric_limits<double>::max();
+  double maxCoord = std::numeric_limits<double>::lowest();
+  QMap< double, QgsLayoutItem * > itemCoords;
+
+  double step;
+
+  // equispaced distribution doesn't follow the same approach of the other distribution types
+  if ( distribution == DistributeHSpace || distribution == DistributeVSpace )
+  {
+    double length = 0.0;
+
+    for ( QgsLayoutItem *item : items )
+    {
+      QRectF itemBBox = item->sceneBoundingRect();
+      double item_min, item_max;
+
+      item_min = ( distribution == DistributeHSpace ? itemBBox.left() :
+                   itemBBox.top() );
+      item_max = ( distribution == DistributeHSpace ? itemBBox.right() :
```

Comment:
```suggestion
      double itemMax = ( distribution == DistributeHSpace ? itemBBox.right() :
```

QGIS code convention is camel case for variable names